AppointmentDetails.cs

```diff
@@ -1,78 +1,32 @@
-using System;
-using System.Collections.Generic;
 using System.Globalization;
-using System.Linq;
 using System.Text.RegularExpressions;
-using System.Threading.Tasks;
+using Newtonsoft.Json;
 
-namespace TwilioSMSScheduler
-{
-    public class AppointmentDetails
-    {
-        public string summary;
-        public string description;
-        public int colorId;
-        public CalendarDateTime start;
-        public CalendarDateTime end;
-        public AppointmentDetails(string smsBody, string timeZone, string senderPhoneNo)
```

Comment:
It's better to extract the logic into a separate class and leave this as a POCO class.